README.md

```diff
@@ -21,7 +21,7 @@ This package is released under the Apache 2.0 license. See the LICENSE file.
 
 This package has the following requirements:
  - a recent release of scikit-learn. Release 0.17 has been tested, older versions may work too.
- - Spark >= 1.5. Spark may be downloaded from the [Spark official website](http://spark.apache.org/). In order to use this package, you need to use the pyspark interpreter or another Spark-compliant python interpreter. See the [Spark guide](https://spark.apache.org/docs/latest/programming-guide.html#overview) for more details.
+ - Spark >= 2.0. Spark may be downloaded from the [Spark official website](http://spark.apache.org/). In order to use this package, you need to use the pyspark interpreter or another Spark-compliant python interpreter. See the [Spark guide](https://spark.apache.org/docs/latest/programming-guide.html#overview) for more details. NOTICE: currently, this package uses the nightly 2.0.0 snapshot, available [here](http://people.apache.org/~pwendell/spark-nightly/spark-branch-2.0-bin/latest/) (TODO: remove reference after 2.0.0 release).
```

Comment:
Mention that Spark 2 is required for the development version. Can you also move the pypi paragraph above? This should be the main way of using it anyway.